sdk/android/skin/src/main/java/com/ooyala/android/skin/notification/NotificationHandler.java

```diff
@@ -0,0 +1,4 @@
+package com.ooyala.android.skin.notification;
+
+public interface NotificationHandler<T> {
```

Comment:
Do we need empty interface?

Reply:
Yep, valid notice, forgot to remove it.
Done.